DESCRIPTION

```diff
@@ -29,6 +29,24 @@ Suggests:
     plyr,
     tidyr,
     testthat,
-    dataRetrieval,
-    mda.streams,
-    powstreams
\ No newline at end of file
+    dataRetrieval
+Collate:
```

Comment:
i didn't write it. but i did add a @include so that metab_model.R would be run before metab_mle.R, because mle_model inherits from metab_model. my guess is roxygen2 did that for us